ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConstants.java

```diff
@@ -63,7 +64,8 @@ private KsqlConstants() {
   }
 
   public static KsqlQueryStatus fromStreamsState(final KafkaStreams.State state) {
-    return state == KafkaStreams.State.ERROR ? KsqlQueryStatus.ERROR : KsqlQueryStatus.RUNNING;
+    return state == State.ERROR || state == State.CREATED 
```

Comment:
very nit: maybe add a comment explaining why we map CREATED as ERROR as well, for future readers who are not aware of this PR.

Reply:
not updating this mapping anymore